CakeMap.Rmd

```diff
@@ -95,6 +100,10 @@ difficult to provide general rules of thumb regarding simplicity other than
 'do not over-complicate the model with excessive constraint variables and
 constraints'.
 
+So, we always need to reach an equilibrium between these principles. 
```

Comment:
Ok, what word can I use to express an implication? Because I don't know other words to tell "this implies ..."

Le Mardi 20 Janvier 2015 14:06 CET, Robin <notifications@github.com> a écrit: 
 
> > @@ -95,6 +100,10 @@ difficult to provide general rules of thumb regarding simplicity other than
> >  'do not over-complicate the model with excessive constraint variables and
> >  constraints'.
> >  
> > +So, we always need to reach an equilibrium between these principles. 
> 
> You start many sentences with "So, ". This sounds a little too informal to me...
> 
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/Robinlovelace/spatial-microsim-book/pull/20/files#r23220380